nursery/persist-via-application-shimming.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: persist via application shimming
+    namespace: persistence/file-system-and-registry
+    authors:
+      - j.j.vannielen@utwente.nl
+    scopes:
+      static: function
+      dynamic: call
+    att&ck:
+      - Persistence::Event Triggered Execution::Application Shimming [T1546.011]
+    references:
+      - https://cloud.google.com/blog/topics/threat-intelligence/fin7-shim-databases-persistence/
+  features:
+    - or:
+      - and:
+        - match: set registry value
+        - string: /Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\InstalledSDB\\/i
+        - string: /DatabasePath/i
+      - and:
+        - or:
+          - match: copy file
+          - match: move file
+          - match: host-interaction/file-system/write
+        - string: /.sdb/i
```

Comment:
I'm not sure about these features, do you have an example sample handy?

Reply:
The idea behind this part is that malware can look for existing shim files and overwrite them.

Reply:
I see, would you please add a comment to this part directly to the rule?